src/umpire/strategy/AllocationTracker.cpp

```diff
@@ -20,13 +20,23 @@ namespace strategy {
 AllocationTracker::AllocationTracker(
   const std::string& name,
   int id,
-  Allocator allocator) noexcept :
+  Allocator allocator,
+  bool own) noexcept :
 AllocationStrategy(name, id),
 mixins::Inspector(),
+m_owns_allocator(own),
```

Comment:
Can you back this change out?

Reply:
Yeah the destructor no longer gets called so it's not necessary. I'll remove it.